R/font-awesome.R

```diff
@@ -4,7 +4,6 @@ font_awesome_brands <- c(
   "accusoft",
   "acquisitions-incorporated",
   "adn",
-  "adobe",
```

Comment:
What happens when an icon is asked for an it doesn't exist?

Reply:
Looks like it'll generate the HTML, but it obviously won't display in the web page.

```
as.character(shiny::icon("foo", lib = "font-awesome"))
#> [1] "<i class=\"fa fa-foo\" role=\"presentation\" aria-label=\"foo icon\"></i>"
```

The auto-generated file is only used to figure out whether to use a `fab` prefix:

https://github.com/rstudio/shiny/blob/a6347341e3968a1b7b66f840efbc606e673c1a40/R/bootstrap.R#L1546-L1548